kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -525,6 +532,7 @@ def _init_and_run_proof(_init_problem: tuple[str, str]) -> bool:
                 kore_rpc_command=kore_rpc_command,
                 smt_timeout=smt_timeout,
                 smt_retry_limit=smt_retry_limit,
+                failure_info=failure_info,
                 trace_rewrites=trace_rewrites,
             )
```

Comment:
I guess we need to do something like:

```
result = kevm_apr_prove(...)
if result is failing:
    print extra foundry information also show in `show`
return result
```

So maybe I'd factor out a helper `Foundry.help_info`, and just print that in both places.

Reply:
I wasn't sure I fully understood what you wanted here. I factored out the repeated code into a `help_info` method and call this in `init_and_run_proof` instead of in `kevm_apr_prove`. Does this address the issue?